modules/tables/table_NUM_Pranavnk15.html

```diff
@@ -0,0 +1,96 @@
+<div>
+    <style>
+        #table-content {
+            padding-top: 10%;
+            text-align: center;
+        }
+
+        table {
```

Comment:
Use precise class names so that your module does not affect another module or outer elements using this tag.